src/test/java/teammates/test/driver/TestUrl.java

```diff
@@ -0,0 +1,25 @@
+package teammates.test.driver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import teammates.common.util.Url;
+
+/**
+ *TestUrl.
```

Comment:
If extending from `AppUrl` class, it can be `Additional AppUrl utility methods used only in tests.`